Aztec/Classes/TextKit/ParagraphProperty/TextList.swift

```diff
@@ -14,10 +14,19 @@ open class TextList: ParagraphProperty {
         case ordered
         case unordered
 
-        func markerText(forItemNumber number: Int) -> String {
+        func markerText(forItemNumber number: Int, indentLevel: Int = 1) -> String {
             switch self {
-            case .ordered:      return "\(number)."
-            case .unordered:    return "\u{2022}"
+            case .ordered:
+                return "\(number)."
+            case .unordered:
+                switch indentLevel {
+                case 1:
+                    return "\u{2022}"
+                case 2:
+                    return "\u{2E30}"
+                default:
+                    return "\u{2B29}"
+                }
```

Comment:
This change is implemented in a way that it changes the default bullet look for any App using the editor.  Additionally the change makes it impossible to go back to the previous behavior.

We'd need an alternative way to make this not override the existing default.

Reply:
Understood. I have a solution.

Reply:
I provided a solution: `TextList.IndentStyle`